buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1258,23 +1258,20 @@ class BeamModulePlugin implements Plugin<Project> {
         exclude group: "org.hamcrest", module: "hamcrest-all"
       }
 
-      // Force usage of the libraries defined within our common set found in the root
+      // For tests, force usage of the libraries defined within our common set found in the root
       // build.gradle instead of using Gradles default dependency resolution mechanism
       // which chooses the latest version available.
       //
       // TODO: Figure out whether we should force all dependency conflict resolution
-      // to occur in the "shadow" and "shadowTest" configurations.
+      // to occur in the "shadowTest" configurations.
       project.configurations.all { config ->
-        // When running beam_Dependency_Check, resolutionStrategy should not be used; otherwise
-        // gradle-versions-plugin does not report the latest versions of the dependencies.
-        def startTasks = project.gradle.startParameter.taskNames
-        def inDependencyUpdates = 'dependencyUpdates' in startTasks || 'runBeamDependencyCheck' in startTasks
-
-        // The "errorprone" configuration controls the classpath used by errorprone static analysis, which
-        // has different dependencies than our project.
-        if (config.getName() != "errorprone" && !inDependencyUpdates) {
```

Comment:
Before this PR, the effect of `force` was unclear; this file had to special-case "errorprone" and "runBeamDependencyCheck" when unexpected behavior happened.